src/screens/TodayScreen.js

```diff
@@ -20,20 +19,11 @@ class TodayScreen extends Component {
     }
 
     componentDidMount() {
-        // const moodTemplate = fetch('http://localhost:5000/mood_templates/5d86024b-1bae-42f1-8a2c-028fe18e33e0')
-        //     .then((response) => response.json())
-        //     .then((responseJson) => {
-        //         console.log(responseJson);
-        //             this.setState({
-        //                 moodTemplate: responseJson.data,
-        //             }); 
-        //     })
-        //     .catch((error) => {
-        //         console.log(error);
-        //     });
-
+        if (!global.useCustomUrl) {
+            global.customUrl = '127.0.0.1:5000';
+        }
         //todo api needs updating to enable selecting all the moods from a particular mood template
-        return fetch('http://127.0.0.1:5000/moods/')
+        return fetch('http://' + global.customUrl + '/moods/')
```

Comment:
do you supply the custom URL without the http/https? I think that would seem weird to most people supplying the url without. maybe we could detect whether they had supplied https or not and add it for them.